.github/workflows/conformance-kind.yaml

```diff
@@ -124,7 +124,7 @@ jobs:
           cilium connectivity test --force-deploy --flow-validation=disabled
 
       - name: Post-test information gathering
-        if: ${{ always() }}
+        if: ${{ !success() }}
```

Comment:
Was this `always()` on purpose?

Reply:
No, I simply missed it back in #16010.

Reply:
That said we can stick to `always()` for the Kind workflow if we feel it's useful, it's not really an issue considering its specificities (runs on `pull_request`, fast, can be pre-empted without issues). I changed it in this PR only out of consistency and because I didn't think it was useful.